test-apps/display-test-app/vite.config.ts

```diff
@@ -19,6 +19,18 @@ const mode =
 
 // array of public directories static assets from dependencies to copy
 const assets = ["./public/*"]; // assets for test-app
+const itwinPackagesWithoutAliases = [
+  "@itwin/browser-authorization",
+  "@itwin/core-backend",
+  "@itwin/editor-backend",
+  "@itwin/electron-authorization",
+  "@itwin/imodels-access-backend",
+  "@itwin/imodels-access-frontend",
+  "@itwin/imodels-client-authoring",
+  "@itwin/imodels-client-management",
+  "@itwin/object-storage-core",
+  "@itwin/reality-data-client"]; // backend packages and packages not in itwin core, which are not included in the vite bundle or do not have frontend .ts sourcemaps and thus do not need aliases
```

Comment:
bad formatting
comment should be above the the var declaration

we shouldnt need to list all of these pkgs; require.resolve should be able to help you determine if the pkg is within the monorepo or not